### ISSUE TITLE
Redmi S2 missing touchscreen firmware

### ISSUE THREAD

**Issue by MacTavishAO** [MiCode/Xiaomi_Kernel_OpenSource#930]
Xiaomi, you are releasing kernel rarely at least do it RIGHT!

`fatal error: include/firmware/LQ_ysl_FT5446_Ofilm_EBBG_V0d_D01_20180403_app.i: No such file or directory
`

**Comment by muralivijay** [comment]
lol xiaomi devs . idk why they always missing device important drivers . before redmi 5A battery driver issue . but now redmi s2 aka ysl focaltech firmware  driver missing . :D :D

**Comment by tonystark009** [comment]
yes why are you cheating xiaomi, after became no.1 company head weight came uh??? What the hell!!! Dont chest karma is there

**Comment by MacTavishAO** [comment]
Still don't give a shit ?

**Comment by luk1337** [comment]
correct.

**Comment by sushant9096** [comment]
drivers/input/touchscreen/focaltech_touch/focaltech_flash.c:50:28: fatal error: include/firmware/LQ_ysl_FT5446_Ofilm_EBBG_V0d_D01_20180403_app.i: No such file or directory
 #include FTS_UPGRADE_FW_APP
Xiaomi why you are dropped this firmware from source

**Comment by dazza5000** [comment]
Did you find the firmware?

**Comment by dazza5000** [comment]
I found the firmware in a kernel dump from a lenovo tb8504f
